src/www/ui_components/app-root.js

```diff
@@ -540,6 +540,10 @@ export class AppRoot extends mixinBehaviors([AppLocalizeBehavior], PolymerElemen
       toastUrl: {
         type: String,
       },
+      showAltAccessMessage: {
+        type: Boolean,
+        computed: '_computeShowAltAccessMessage()',
```

Comment:
You need to express the dependency on the language so it gets recomputed on change.
 
```suggestion
        computed: '_computeShowAltAccessMessage(language)',
```

Reply:
Please test the behavior change on language change.

Reply:
done, this definitely fixed some of the weirdness on language change